README.md

```diff
@@ -139,18 +159,28 @@ $ sw-precache --root=dist --static-file-globs='dist/**/*.html'
 
 The `sw-precache` module exposes two methods: `generate` and `write`.
 
-#### generate(options, callback)
-`generate` takes in [options](#options), generates resulting service worker code as a string, and
-then invokes `callback(error, serviceWorkerString)`.
-In the 1.x releases of `sw-precache`, this was the default and only method exposed by the module.
+#### generate(options, callback) `generate` takes in [options](#options),
```

Comment:
That was my editor's wrap command taking undue liberties. I've fixed both
generate() and write().

Joe Medley | Technical Writer, DevPlat | jmedley@google.com | 816-678-7195

On Tue, Nov 10, 2015 at 1:56 PM, Jeffrey Posnick <notifications@github.com>
wrote:

> In README.md
> <https://github.com/GoogleChrome/sw-precache/pull/50#discussion_r44471264>
> :
>
> > @@ -139,18 +159,28 @@ $ sw-precache --root=dist --static-file-globs='dist/**/*.html'
> >
> >  The `sw-precache` module exposes two methods: `generate` and `write`.
> >
> > -#### generate(options, callback)
> > -`generate` takes in [options](#options), generates resulting service worker code as a string, and
> > -then invokes `callback(error, serviceWorkerString)`.
> > -In the 1.x releases of `sw-precache`, this was the default and only method exposed by the module.
> > +#### generate(options, callback) `generate` takes in [options](#options),
>
> This Markdown leads to what I assume is unintentional formatting:
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/1749548/11076958/a5b384b0-87cb-11e5-9792-a0cd56a6e6f9.png>
>
> In general, can you make sure you View the rendered Markdown using the
> GitHub web interface and check for problematic formatting? There are a few
> other places in this file that show similar issues (first lines of text
> being included in headings, for example). I can point them out one-by-one,
> but I think they'll stand out if you use the View option.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/sw-precache/pull/50/files#r44471264>.
>